src/promptflow/tests/test_configs/eager_flows/basic_callable_class_with_sample_file/simple_callable_class.py

```diff
@@ -1,11 +1,14 @@
 # ---------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
-from typing import TypedDict
+
+from dataclasses import dataclass
 
 from promptflow.tracing import trace
 
-class FlowOutput(TypedDict):
+
+@dataclass
```

Comment:
Consider add a new test case for dataclass output? We need to support both TypedDict and dataclass.

Reply:
Will fix in other PR.